src/controller/autotrack/Helpers/AutoTrackerSettings.py

```diff
@@ -0,0 +1,100 @@
+import logging

```

Comment:
nur "from logging import getLogger"
 importieren das macht es einfacher das nicht ausversehen doch logging.error() oder so aufgrufen wird

Reply:
This is now fixed.